src/AddTodoForm.js

```diff
@@ -1,13 +1,23 @@
 import React from 'react';
 
-const FORM_ID = 'todoTitle'
+const FORM_ID = 'todoTitle';
 
-const AddTodoForm = () => (
-        <form>
-            <label htmlFor={FORM_ID}>Title: </label>
-            <input id={FORM_ID}/>
-            <button type={'submit'}>Add</button>
+const AddTodoForm = ({ onAddToDo }) => {
+
+    const handleAddTodo = (event) => {
+        event.preventDefault();
+        onAddToDo(event.target.title.value);
+        event.target.title.value = '';
```

Comment:
Hey Matt, thanks as always for the great suggestions, I think I've fixed everything you suggested, as well as the required change. This is what this page looks like now. Hopefully I did it correctly, it's still functioning, so that's a good sign.

I was just wondering if I need to submit another pull request and/or redo the assignment form.

```javascript
import React from 'react';

const TITLE_ID = 'todoTitle';
const FORM_ID = 'addTodoForm';

const AddTodoForm = ({ onAddToDo }) => {

    const handleAddTodo = (event) => {
        event.preventDefault();
        const todoTitle = event.target.title.value;
        onAddToDo(todoTitle);
        document.querySelector(`#${FORM_ID}`).reset();
    };

    return (
        <form id={FORM_ID} onSubmit={handleAddTodo}>
            <label htmlFor={TITLE_ID}>Title:&nbsp;
                <input id={TITLE_ID} name='title'/>
                <button type='submit'>Add</button>
            </label>
        </form>
    );
};

export default AddTodoForm;
```

Reply:
No need to submit another PR! Under the reviewers section in the PR, click the button that looks like a refresh icon. That’ll mark the PR as requesting another review from me!

Reply:
Ok thanks! I just pushed the changed, so you should be able to see them hopefully.